tests/test_pagination.py

```diff
@@ -0,0 +1,93 @@
+from tap_tester import runner, connections
+from base import SnapchatBase
+
+class SnapchatPaginationTest(SnapchatBase):
+
+    page_size = 1
+
+    def name(self):
+        return "tap_tester_snapchat_ads_pagination_test"
+
+    def get_properties(self, *args, **kwargs):
+        props = super().get_properties(*args, **kwargs)
+        # add page size of 1
+        props['page_size'] = self.page_size
+        return props
+
+    def test_run(self):
+        # we are getting duplicate records for 'id' fields for this stream
+        # when asked support about this, but this is known behavior from the API side
+        # Please refer card: https://jira.talendforge.org/browse/TDL-18686 for more details
+        known_failing_streams = {"targeting_android_versions"}
+
+        # phone_numbers: need to another phone number and verify it using OTP
+        # pixels: not able to generate another record
+        stream_to_skip = {"phone_numbers", "pixels"}
+
+        pagination_not_supported_fields = {"funding_sources", "members", "ad_account_stats_daily", "ad_account_stats_hourly", "pixel_domain_stats", "campaign_stats_daily", "campaign_stats_hourly", "ad_squad_stats_daily", "ad_squad_stats_hourly", "ad_stats_daily",  "ad_stats_hourly"}
```

Comment:
Update variable name to `pagination_not_supported_streams` and also add comments about all the maps defined in the test.

Reply:
Added.